manpages/brew.1

```diff
@@ -3240,6 +3240,12 @@ If set, always assume \fB\-\-verbose\fR when running commands\.
 If set, always assume \fB\-\-debug\fR when running commands\.
 .
 .TP
+\fBHOMEBREW_SEQUENTIAL_FETCH\fR
+.
+.br
+If set, use sequential git fetching instead of fetching in parallel\.
```

Comment:
I think it could be helpful to reference why this would be used. Keeping this easier to debug and resolve.